Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nvcr.io/nvidia/pytorch:24.07-py3
```

Comment:
[PyTorch Release 24.07](https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel-24-07.html)

Reply:
This is what NeMo uses in their [Dockerfile.ci](https://github.com/NVIDIA/NeMo/blob/main/Dockerfile.ci#L17) file. Happy to use a different version if people have other preferences.

Reply:
My preference is RAPIDS CI container too.  We are not really tip of the spear when it comes to using Pytorch features but are for RAPIDS so probably using a barebones Rapids CI container will be better to make it similar to other RAPIDS builds.

Reply:
Thanks all! Updated.